common/lib/capa/capa/inputtypes.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, status, gettext_func=unicode):
         names = {
             'correct': _('correct'),
             'incorrect': _('incorrect'),
+            'partially-correct': _('partially-correct'),
```

Comment:
We only internationalize user facing text, so either this is user facing
and should be "partially correct", or it is not user facing and should not
be internationalized.

On Thu, Jun 25, 2015 at 4:54 PM, Colin-Fredericks <notifications@github.com>
wrote:

> In common/lib/capa/capa/inputtypes.py
> <https://github.com/edx/edx-platform/pull/8574#discussion_r33303708>:
>
> > @@ -85,6 +85,7 @@ def __init__(self, status, gettext_func=unicode):
> >          names = {
> >              'correct': _('correct'),
> >              'incorrect': _('incorrect'),
> > +            'partially-correct': _('partially-correct'),
>
> It's probably not. I thought it was turning problem status into a CSS
> class name, but there's something I don't understand here, because changing
> the right-hand text doesn't change the name of the CSS class. I'll keep
> poking at this intermittently.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8574/files#r33303708>.
>